gr-uhd/lib/usrp_block_impl.cc

```diff
@@ -670,6 +676,18 @@ void usrp_block_impl::_cmd_handler_rate(const pmt::pmt_t& rate_, int, const pmt:
     set_samp_rate(rate);
 }
 
+void usrp_block_impl::_cmd_handler_pc_clock_resync(const pmt::pmt_t&,
+                                                   int,
+                                                   const pmt::pmt_t&)
+{
+    const uint64_t ticks =
```

Comment:
Should we add a warning here? smth like:
```cpp
if(clock_source() != "internal"){
    GR_LOG(WARN, "Clock source mismatch: Syncing USRP to PC clock but USRP is configured to an external source!");
}

Reply:
Yes. This is what I was imagining. Though it'd be Time Source.

> Time source mismatch: Setting USRP Time to PC clock but USRP is configured to use an external PPS. The PPS will be ignored.

Something like this?